firmware/peripherals/Kconfig

```diff
@@ -1,9 +1,4 @@
 menu "Peripherals"
-menu "Hardware Periph Setup"
-if BOARD_M4A_24G
-rsource "../../boards/m4a-24g/Kconfig.periph"
-endif
```

Comment:
This should be integrated to select the ADC_VREF from the board, Here could be added more boards...